nebullvm/inference_learners/tvm.py

```diff
@@ -18,6 +22,7 @@
 )
 from nebullvm.base import ModelParams, DeepLearningFramework
```

Comment:
move this import up

Reply:
solved in https://github.com/nebuly-ai/nebullvm/pull/74/commits/892c91bb8001698f6eb385215f68c3e7f91fff86